manifests/role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: peribolos-role-binding
+  namespace: peribolos
+subjects:
+- kind: ServiceAccount
+  name: peribolos-as-service
+  namespace: peribolos
```

Comment:
Kustomize can also change this value. Let's leave the namespace here to be filled in as well...